app/Http/Controllers/Auth/AuthController.php

```diff
@@ -48,6 +49,7 @@ public function register(RegisterRequest $registerRequest)
         $user = $this->userRepository->create($data);
 
         if ($user) {
+            AssignStarterProjects::dispatch($user, $user->createToken('auth_token')->accessToken)->delay(now()->addSecond())->onQueue('starterProjects');
```

Comment:
Why do we need create token here?

Reply:
The clone function which I'm using is created by mahmod and it requires users token to complete the clone request.

Reply:
How does it work when user login and generate new token?

Reply:
When user created from admin panel, I create the token so I can pass the required data to mahmood's function for assignment of starter projects.

Reply:
It's not in admin registration. It's normal registration, isn't it?

Reply:
Yes, normal user registration from admin panel.

Reply:
So how does it work when user login and generate new token?

Reply:
Token will be generated whenever there is a clone request for any user.

Reply:
No login required for generation of token. Once user is created, created access token on that new user model object.

Reply:
@ahmad-curriki  @yurysakovich1010  - at the moment, Laravel passport is allowing multiple access tokens for the user, and the old token is not revoked. Ahmad has tested this, so we can merge it for now, but it was a good catch by Yury, and that use case had to be tested.
For now, we can merge.